lib/qiita/markdown/filters/greenmat.rb

```diff
@@ -16,13 +20,18 @@ def greenmat
             Qiita::Markdown::Greenmat::HTMLRenderer.new(hard_wrap: true, with_toc_data: true),
             autolink: true,
             fenced_code_blocks: true,
-            footnotes: true,
+            footnotes: options[:footnotes],
             no_intra_emphasis: true,
             no_mention_emphasis: true,
             strikethrough: true,
             tables: true,
           )
         end
+
+
+        def options
+          @options ||= DEFAULT_OPTIONS.merge(context[:greenmat] || {})
```

Comment:
I think the context key should be something more abstract (such as `:markdown`) so that we can replace the internal Markdown renderer from Greenmat to another renderer without breaking this context key compatibility in the future. The key `:greenmat` is an internal implementation thing.